boards/sparkfun/thing_plus_matter_mgm240p/sparkfun_thing_plus_matter_mgm240p.dts

```diff
@@ -53,6 +54,17 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+&eusart1 {
```

Comment:
Nitpick, we usually keep a blank line between the declaration blocks.

Reply:
Fixed, thanks for your comment!